src/NHPersistenceCompatibilityTests/Common/NHibernateConnectionInfo.cs

```diff
@@ -4,8 +4,8 @@ namespace Common
 {
     public class NHibernateConnectionInfo
     {
-        const string Dialect = "NHibernate.Dialect.MsSql2012Dialect";
-        const string ConnectionString = @"Data Source=localhost\SQLEXPRESS;Initial Catalog=persistencetests;Integrated Security=True";
+        public const string Dialect = "NHibernate.Dialect.MsSql2012Dialect";
+        public const string ConnectionString = @"Data Source=localhost\SQLEXPRESS;Initial Catalog=persistencetests;Integrated Security=True";
```

Comment:
@tmasternak I think Ramon means putting it in `ConnectionStrings` section of the config file.